docs/blog.md

```diff
@@ -270,7 +270,7 @@ The `subscribe` method returns the `unsubscribe` method, but in my case I segreg
 
 #### Sender
 
-Here is where the polling runs and does batch calling to all the token contracts and sends to the waku topic that we reated.
+Here is where the polling runs and does batch calling to all the token contracts and sends to the Waku topic that we ??reated??.
```

Comment:
I was not sure what `reated` means or which word was it supposed to be:)

Reply:
😅 its `created`.